### PR TITLE
Monolog symfony example

### PR DESCRIPTION
**Pull request by krcrawford**
In this PR, I've updated some of the docs to include a Monolog usage example for Symfony applications.

### CODE REVIEW

**Comment by krcrawford** [comment]
@GromNaN Fair enough. This was really in response to the [README.md](https://github.com/mnapoli/bref/blob/master/README.md) as requested here:
>If you are using Monolog this means you will need to configure Monolog to write to the output (contribution welcome: clarify with an example).

I agree that the minimalist logger would be a leaner solution, as you suggest.

@mnapoli Do you want to continue providing docs that suggest the use of Monolog? I may introduce another PR around removing the Monolog suggestion and suggesting the minimalist logger (unless @GromNaN creates a PR)

File: docs/Symfony.md [hunk 1]
```diff
@@ -87,6 +87,50 @@ public function getLogDir()
 
 The best solution however is not to write log on disks because those are lost. You should use a remote log collector (ELK stack) or a cloud solution like Cloudtrail, Papertrail, Loggly, etc.
 
+While `stdout` and `stderr`echo statements get logged directly to CloudWatch, you can easily use Monolog to write to CloudWatch using `php://stderr`.
```
Comment (staabm) [inline docs/Symfony.md]: Missing space before „echo“

File: docs/Symfony.md [hunk 2]
```diff
@@ -87,6 +87,50 @@ public function getLogDir()
...
+
+```shell
+composer require symfony/monolog-bundle
+```
+
+```yaml
+#config/packages/prod/monolog.yaml
```
Comment (staabm) [inline docs/Symfony.md]:
```suggestion
# config/packages/prod/monolog.yaml
```

**Comment by mnapoli** [comment]
>  I may introduce another PR around removing the Monolog suggestion and suggesting the minimalist logger instead

That sounds even better indeed! If you can do that that's awesome thanks!

**Comment by mnapoli** [comment]
Closing this PR in the meantime.

**Pull request closed without merging**